### DISCUSSION TITLE
LDN Rule-Number of Copies

### DISCUSSION THREAD

**Discussion by Sunnyiscoming** [filecoin-project/notary-governance#834] [General]
There is no specific rule on the number of duplicates. It mostly exists in the following two situations.

1. Multiple requests for the same open dataset, such as
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1587
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1579

2. Number of copies in one/same application, such as
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1266
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1267
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1268

There are some comments in the community about this issue. Hope to receive some more outstanding thoughts.

**Comment by kernelogic** [comment]
See my opinions:
https://github.com/filecoin-project/notary-governance/issues/832#issuecomment-1441243099
https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1545#issuecomment-1440326173

TLDR: I support multiple requests per dataset.

If fil+ team decides to reject these scenarios, a lot current LDNs need to be revoked.
